src/index.js

```diff
@@ -37,7 +38,7 @@ window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){va
             { type: 'text/javascript' },
             this.newrelicstring,
           );
-          data.bodyTags.push(newRelicScriptTag);
+          data.headTags.push(newRelicScriptTag);
```

Comment:
New Relic instructions read: 
> Copy the snippet of code below and paste it as close to the top of the HEAD as possible, but after any position-sensitive META tags (X-UA-Compatible and charset).

Does the README need any special instructions for ensuring it is inserted in the right place?

Reply:
As discussed in #4  we need to ensure that the script is placed not only in the head tag but, after the meta tags and before all other scripts. I'll think about it and figure this out as soon as possible.